build.sbt

```diff
@@ -899,7 +901,18 @@ lazy val queryableState = (project in engine("queryableState")).
     }
   ).dependsOn(api)
 
-
+lazy val sql = (project in component("sql")).
+  configs(IntegrationTest).
+  settings(commonSettings).
+  settings(Defaults.itSettings).
+  settings(
+    name := "nussknacker-sql",
+    libraryDependencies ++= Seq(
+      "com.zaxxer" % "HikariCP" % "4.0.3",
+      "org.apache.flink" %% "flink-streaming-scala" % flinkV % Provided,
```

Comment:
let's add it as optional, so that these services can be used also in standalone

Reply:
done